packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/entity-events-to-db.listener.ts

```diff
@@ -58,6 +65,7 @@ export class EntityEventsToDbListener {
 
     this.messageQueueService.add<SaveEventToDbJobData>(SaveEventToDbJob.name, {
       workspaceId: payload.workspaceId,
+      userId: payload.userId,
```

Comment:
Shouldn't we use the workspaceMemberId instead?

Reply:
We don't have access to the workspaceMemberId easy in that context, that's why I retrieve it later